integration-tests/narayana-jta/src/test/java/io/quarkus/narayana/jta/TransactionJdbcObjectStoreTest.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.narayana.jta;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.common.http.TestHTTPEndpoint;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+
+@QuarkusTest
+@TestHTTPEndpoint(TransactionalResource.class)
+@TestProfile(JdbcObjectStoreTestProfile.class)
```

Comment:
@mmusgrov We use `JdbcObjectStoreTestProfile` here to active those properies to test with jdbc object-store.

Reply:
Well, I think it is not related.